openshift/keycloak.app.yaml

```diff
@@ -103,6 +109,14 @@ objects:
               secretKeyRef:
                 name: keycloak-config
                 key: db.port
+          - name: OPENSHIFT_KUBE_PING_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+          - name: OPENSHIFT_KUBE_PING_LABELS
+            value: 'name=keycloak-server'
+          - name: OPENSHIFT_KUBE_PING_SERVER_PORT
+            value: '47600'
```

Comment:
Can we add an ENV variable with the CLUSTERED mode `standalone` to be more explicit ?